fedimint-server/src/config/mod.rs

```diff
@@ -403,6 +418,14 @@ impl ServerConfig {
         let mut rng = OsRng;
         let peer0 = &params[&PeerId::from(0)];
 
+        let mut broadcast_pks = BTreeMap::new();
+        let mut broadcast_sks = BTreeMap::new();
+        for peer_id in peer0.peer_ids() {
+            let (broadcast_sk, broadcast_pk) = secp256k1_zkp::generate_keypair(&mut OsRng);
+            broadcast_pks.insert(peer_id, broadcast_pk);
+            broadcast_sks.insert(peer_id, broadcast_sk);
+        }
```

Comment:
This code is perfectly fine, just FYI that this could also be achieved with less mutable state (didn't compile it, just a sketch):

```rust
let (broadcast_pks, broadcast_sks) = peer0.peer_ids()
    .into_iter()
    .map(|peer_id| {
        let (broadcast_pk, broadcast_sk) = secp256k1_zkp::generate_keypair(&mut OsRng);
        ((peer_id, broadcast_pk), (peer_id, broadcast_sk))
    })
    .unzip::<BTreeMap<_, _>, BTreeMap<_, _>, _, _>();
```